### ISSUE TITLE
enter-key breaks code in demo editor

### ISSUE THREAD

**Issue by Engelberg** [parinfer/parinfer.js#149]
```
(defn foo
  "hello, this is a docstring"
  [a b]
  (let [sum (+ a b)
        prod (* a b)]
     {:sum sum
      :prod prod}))
```

Put cursor before the left square bracket in the let and hit enter. It changes your code entirely.

**Comment by shaunlebron** [comment]
This is a problem in the [demo editor](http://shaunlebron.github.io/parinfer/demo).  My integration with Code Mirror seems to have broken last week.  Will fix

**Comment by shaunlebron** [comment]
I started a new version of Parinfer for CodeMirror here:
https://github.com/shaunlebron/parinfer-codemirror

The problem with pressing enter is fixed there, but there's more work that needs to be done before I can replace the demo editor with it.

**Comment by shaunlebron** [comment]
will be fixed in #155

**Comment by shaunlebron** [comment]
@Engelberg fixed in latest [demo editor](http://shaunlebron.github.io/parinfer/demo). thanks
